web/src/routes/LicensePage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { License, parseLicenses } from 'api-client/src/utils/licences'
+
+import Caption from '../components/Caption'
+import List from '../components/List'
+import ListItem from '../components/ListItem'
+import { reportError } from '../utils/sentry'
+
+const LicenseContainer = styled.div`
+  padding: 5px 15px;
+  line-height: 120%;
+`
+
+type PropsType = {
+  name: string
+  version: string | undefined
+  license: string
+  onPress: string
+}
+
+const LicenseItem = (props: PropsType): ReactElement => {
+  const { name, version, license, onPress } = props
+  return (
+    <ListItem path={onPress} title={name}>
+      <LicenseContainer>
+        {`version: ${version}`}
+        <br />
+        {`license: ${license}`}
+      </LicenseContainer>
+    </ListItem>
+  )
+}
+
+const LicensePage = (): ReactElement => {
+  const { t } = useTranslation('settings')
+  const [licenses, setLicenses] = useState<License[]>([])
+
+  useEffect(() => {
+    import('../../assets/licenses.json')
+      .then(licenseFile => setLicenses(parseLicenses(licenseFile.default)))
+      .catch(error => reportError(`error while importing licenses ${error}`))
+  }, [])
+
+  const renderItem = (item: License) => (
+    <LicenseItem name={item.name} license={item.licenses} version={item.version} onPress={item.licenseUrl} />
+  )
+
+  return (
+    <div>
+      <Caption title={t('openSourceLicenses')} />
+      <List items={licenses} renderItem={renderItem} noItemsMessage='TODO' />
```

Comment:
Should we add an extra translation for "no licenses found" or should we just use an english text without translation? The licenses are english anyway

Reply:
i'm fine with english only.

Reply:
i think we should keep consistent and add a translation

Reply:
It should never be shown since there will always be more than zero licenses. So I would be fine with just passing an empty string as well.

Reply:
good idea